examples/counter_in/read_freq.py

```diff
@@ -0,0 +1,27 @@
+"""Example of counter input frequency acquisition.
+
+This example demonstrates how to measure frequency using one
+counter on a Counter Input Channel. The Edge, Minimum Value and
+Maximum Value are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import Edge, FrequencyUnits
+
+
+with nidaqmx.Task() as task:
+    channel = task.ci_channels.add_ci_freq_chan(
+        "Dev1/ctr0",
+        min_val=100.0,
+        max_val=1000.0,
```

Comment:
100-1000 seems pretty specific, I would prefer something like 2-1000 or even 2-100000 (if that works).

Same for read_pulse_freq.py

The C example's default range is kind of weird, I wouldn't copy it:
```
DAQmxErrChk (DAQmxCreateCIFreqChan(taskHandle,"Dev1/ctr0","",1.192093,10000000.000000,DAQmx_Val_Hz,DAQmx_Val_Rising,DAQmx_Val_LowFreq1Ctr,0.001,4,""));
```